docs/extended_api/pipelines/pipeline/consumer_wait.md

```diff
@@ -0,0 +1,55 @@
+---
+grand_parent: Pipelines
+parent: cuda::pipeline
+---
+
+# `cuda::pipeline::consumer_wait*`
```

Comment:
Nit: extra `*`

Reply:
Ah, that's intentional; it's meant to be regex style. But maybe that's confusing, I'll change it back.